src/sentry/static/sentry/app/views/organizationIntegrations/installedPlugin.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t} from 'app/locale';
+import Access from 'app/components/acl/access';
+import Button from 'app/components/button';
+import Confirm from 'app/components/confirm';
+import Alert from 'app/components/alert';
+import ProjectBadge from 'app/components/idBadge/projectBadge';
+import withApi from 'app/utils/withApi';
+import {Client} from 'app/api';
+import {
+  addErrorMessage,
+  addSuccessMessage,
+  addLoadingMessage,
+} from 'app/actionCreators/indicator';
+import {PluginNoProject, PluginProjectItem, Organization, AvatarProject} from 'app/types';
+
+export type Props = {
+  api: Client;
+  plugin: PluginNoProject;
+  projectItem: PluginProjectItem;
+  organization: Organization;
+  onResetConfiguration: (projectId: string) => void;
+  onEnablePlugin: (projectId: string) => void;
+  className?: string;
+};
+
+export class InstalledPlugin extends React.Component<Props> {
+  getConfirmMessage() {
+    return (
+      <React.Fragment>
+        <Alert type="error" icon="icon-circle-exclamation">
+          {t(
+            'Deleting this installation will disable the integration for this project and remove any configurations.'
+          )}
+        </Alert>
+      </React.Fragment>
+    );
+  }
+
+  pluginUpdate = async (data: object) => {
+    const {organization, projectItem, plugin} = this.props;
+    await this.props.api.requestPromise(
+      `/projects/${organization.slug}/${projectItem.projectSlug}/plugins/${plugin.id}/`,
```

Comment:
@scefali maybe wrap this in a `try {} catch {}` block.

Reply:
@NisanthanNanthakumar I purposely do not so that the caller function handles the error. See `handleReset` and `enablePlugin`. But I can add a comment explaining this.

Reply:
@scefali I noticed that. If you want to propagate the error then in the `catch`, we can `throw` the error.

Reply:
@NisanthanNanthakumar so just a try/catch and immediately throw the error? Just curious, what's the point of that?